### PR TITLE
Added recipe for podman

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Presently we are only building for linux.  Odds are that we probably
need to add some glibc minimums in here too

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mbargull** [comment]
@mariusvniekerk, FYI, I'm working on the https://github.com/containers stack currently. I have `runc`, `crun`, `fuse-overlays`, `buildah`, and their dependencies built/updated locally (and all built with our CentOS 6 sysroot). I haven't opened staged-recipes PRs for them yet because I wanted to test them out first (meaning, really test, on kinda vanilla VMs to see if they completely work on their own). I'm doing that over the weekend.
If those work, I'll also build `podman` -- if you don't need it right away this week, you could get it "for free" next one.

**Comment by mariusvniekerk** [comment]
@mbargull that sounds awesome!

File: recipes/podman/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+test:
+  commands:
+    - podman -v
+    - "podman run busybox sh -c \"echo hello from the inside; exit 0\""
+
+about:
+  home: https://github.com/containers/podman
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Terraform is a tool for building, changing, and combining infrastructure safely and efficiently.
```
Comment (dhirschfeld) [inline recipes/podman/meta.yaml:46]:
```suggestion
  summary: Podman: A tool for managing OCI containers and pods
```

**Comment by mariusvniekerk** [comment]
@mbargull how fares your recipes?

**Comment by mbargull** [comment]
Hi @mariusvniekerk, sorry for the delay. I have to work out one last kink to make everything "readily" usable (so that all tools use the configs from within the environment). And push out the additions/updates of deps. I'm finishing this today.

**Comment by mbargull** [comment]
> I'm finishing this today.

... today. (Needs more effort wrt understanding Go and build/runtime conf of the project than expected...)

**Comment by mbargull** [comment]
Sorry for the additional delays. I'm waiting for the remaining deps to go in (https://github.com/conda-forge/staged-recipes/pulls/mbargull) and then I add `podman`, `buildah` and update the `skopeo` feedstock (all ready, only need the deps to go live).

**Comment by mbargull** [comment]
Superseded by gh-13115.

**Pull request closed without merging**